lib/logSearch.rb

```diff
@@ -0,0 +1,23 @@
+
+class LogSearch
+
+  def search(log, stringToFind)
+
+    if not File.exist?(log)
```

Comment:
You could consider changing `if not` to `unless`...which do you think is more readable